### PR TITLE
Cherry-pick gitpod #1451 to #1565

### PR DESCRIPTION
**Pull request by MatthewFagan**

### CODE REVIEW

File: components/gitpod-db/src/typeorm/migration/1623652164640-UpdateExamples.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ */
+
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class UpdateExamples1623652164640 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        let priority = 90;
+        const newEntries = [
+            { url: 'https://github.com/gitpod-io/template-typescript-node', description: 'A Node.js app, written in TypeScript.', priority: priority-- },
```
[the remark this answers is not in the document]
Comment (pavan-tri) [reply components/gitpod-db/src/typeorm/migration/1623652164640-UpdateExamples.ts:14]: @MatthewFagan

[the remark this answers is not in the document]
Comment (MatthewFagan) [reply components/gitpod-db/src/typeorm/migration/1623652164640-UpdateExamples.ts:14]: OK @pavan-tri , if you send me the url I can add it to the code

**Review by pavan-tri — COMMENTED** [review]
was download backup tested?

**Comment by MatthewFagan** [comment]
> was download backup tested?

Yes. There seems to be a problem in Chrome, but I think that is a chrome issue. In chrome, you can copy the link and then open the download that way.

**Review by pavan-tri — APPROVED** [review]

**Pull request merged**